sql/moz-fx-data-shared-prod/subscription_platform/apple_subscriptions/view.sql

```diff
@@ -82,10 +81,7 @@ apple_iap_period_aggregates AS (
     periods.start_time,
     periods.end_time,
     periods.period_offset,
-    COALESCE(
-      MAX(events.legacy_subscription_id),
-      periods.original_transaction_id
-    ) AS original_subscription_id,
+    periods.original_transaction_id,
```

Comment:
Downstream CTEs rely on the `original_subscription_id` column.  Simplest solution would be to still output that column here:
```suggestion
    periods.original_transaction_id AS original_subscription_id,
```
Or you could remove this duplicate selection of `original_transaction_id` and change downstream `original_subscription_id` references to `original_transaction_id`.